Image build failed !

Hey,

Thanks for that useful exporter ,i prepared my env to deploy that exporter but looks like the image build failed ! im not a golang guy so im not sure whats wrong .

i will appreciate it if you could guide me to resolve that .

-----------------------> Error : 

```
#10 32.94 collectors/pod_cpu_link.go
#10 32.94 collectors/pod_dev_link.go
#10 32.94 collectors/sriovdev.go
#10 32.94 collectors/sriovdev_readers.go
#10 32.94 pkg/vfstats/netlink.go
#10 32.95 make: golint: No such file or directory
#10 32.95 golint ./...
#10 32.95 make: *** [Makefile:9: build] Error 127
------
executor failed running [/bin/sh -c cd /usr/src/sriov-network-metrics-exporter && make clean && make build]: exit code: 2
```

Regards .

Quick update 

I tried to cd into the dir before running the build and i got this error :


```
 => [internal] load build context                                                                                                                               0.0s
 => => transferring context: 9.54kB                                                                                                                             0.0s
 => CACHED [builder 2/5] RUN apk add --update --virtual build-dependencies build-base linux-headers                                                             0.0s
 => [builder 3/5] ADD ./ /usr/src/sriov-network-metrics-exporter                                                                                                0.0s
 => [builder 4/5] RUN cd /usr/src/sriov-network-metrics-exporter                                                                                                0.3s
 => ERROR [builder 5/5] RUN make clean && make build                                                                                                            0.3s
------
 > [builder 5/5] RUN make clean && make build:
#13 0.241 make: *** No rule to make target 'clean'.  Stop.
------
executor failed running [/bin/sh -c make clean && make build]: exit code: 2
```



Dockerfile change :

```
ADD ./ /usr/src/sriov-network-metrics-exporter

RUN cd /usr/src/sriov-network-metrics-exporter
RUN make clean && make build
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,13 +2,13 @@ FROM golang:alpine as builder
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
-RUN apk add --no-cache --virtual build-dependencies build-base=0.5-r2 linux-headers=5.10.41-r0 git=2.34.2-r0
+RUN apk add --no-cache --virtual build-dependencies build-base linux-headers git
 COPY ./ /usr/src/sriov-network-metrics-exporter
 WORKDIR /usr/src/sriov-network-metrics-exporter
 RUN make clean && make build
 
 FROM alpine:3.16
 COPY --from=builder /usr/src/sriov-network-metrics-exporter/bin/* /usr/bin/
-RUN apk update && apk add --no-cache ca-certificates=20211220-r0 && update-ca-certificates && apk add --no-cache openssl=1.1.1o-r0
+RUN apk update && apk add --no-cache ca-certificates && update-ca-certificates && apk add --no-cache openssl
 EXPOSE 9808
 ENTRYPOINT ["sriov-exporter"]

